src/main/flight/pid.c

```diff
@@ -277,7 +277,24 @@ void pidResetIterm(void)
 void pidUpdateAntiGravityThrottleFilter(float throttle)
 {
     if (pidRuntime.antiGravityMode == ANTI_GRAVITY_SMOOTH) {
-        pidRuntime.antiGravityThrottleHpf = throttle - pt1FilterApply(&pidRuntime.antiGravityThrottleLpf, throttle);
+        // calculate a boost factor for P in the same way as for I when throttle changes quickly
+        // at this point the variable antiGravityThrottleHpf is the lowpass of throttle
+        pidRuntime.antiGravityThrottleHpf = pt1FilterApply(&pidRuntime.antiGravityThrottleLpf, throttle);
```

Comment:
Why don't you use a local float : 
float antiGravityThrottleLpf = pt1FilterApply(&pidRuntime.antiGravityThrottleLpf, throttle);

Reply:
Good idea thanks, I'll do that.

Reply:
Yes please, much less confusing.

Reply:
Thanks, done

Reply:
Hopefully that change is done correctly?